gestalt/vault.py

```diff
@@ -31,6 +31,7 @@ def __init__(self,
                 with role and jwt string from kubernetes
         """
         self._scheme: str = scheme
+        self._run_worker = True
```

Comment:
you might want to create a public `def stop(self)` method that sets this internal variable to `False`, just so that its easier and more intuitive for clients to stop underlying thread polls (i.e. our `clearstreet.config` `Config` class)